pkg/controller/registry/registry_controller_suite_test.go

```diff
@@ -35,6 +36,11 @@ import (
 var cfg *rest.Config
 
 func TestMain(m *testing.M) {
+
+        test.SkipSetupIfShort(m)
```

Comment:
`SkipSetupIfShort` cannot take a `testing.M`, this should not compile.

Aside from that, we should only avoid start the integration control plane in the suite test, but really run `m.Run` so all tests that are not skipped within are actually executed.

Something like: https://github.com/kubic-project/dex-operator/blob/master/pkg/controller/dex/dex_controller_suite_test.go#L40-L60